src/cmd/embeddedrules/rules.php

```diff
@@ -368,3 +368,21 @@ function indexingSyntax() {
      */
     $x{$y};
 }
+
+/**
+ * @comment Report a call define function with third arguments true or false to define case (in)sensitive constants.
+ * @before  define("Z_CONST", 1, true);
+ * @after   define("Z_CONST", 1);
+ */
+function defineConst() {
```

Comment:
```
defineConst -> caseInsensitiveDefine
```

Reply:
Technically, it's also possible to give a case sensitive param. 🤔
Maybe `define3` then?
`defineConst` sounds too generic and it doesn't describe what's the problem.
The problem is 3rd argument.
We could also add this check in `langDeprecated` perhaps.